contrib/cloudflare.service.env

```
@@ -0,0 +1,3 @@
+IP6,IP4
+DOMAINS=<your domains here>
+CF_API_TOKEN=<your cloudflare bearer token>
```

Sorry, some nitpicking:
```suggestion
CF_API_TOKEN=YOUR-CLOUDFLARE-API-TOKEN
DOMAINS=example.org,www.example.org,example.io
```
Reasons: 1. `IP4,IP6` seems to be ill-formed? 2. The tool will check `YOUR-CLOUDFLARE-API-TOKEN` specifically.

---

IP4, IP6 worked on my config, unless it needs changing...I needed both ipv4 and ipv6